flask_restplus_patched/parameters.py

```diff
@@ -72,3 +80,71 @@ def __init__(self, *args, **kwargs):
             self.fields['op'].validators + [validate.OneOf(self.OPERATION_CHOICES)]
         self.fields['path'].validators = \
             self.fields['path'].validators + [validate.OneOf(self.PATH_CHOICES)]
+
+    @validates_schema
+    def validate_patch_structure(self, data):
+        """
+        Common validation of PATCH structure
+
+        Provide check that 'value' present in all operations expect it.
+
+        Provide check if 'path' is present. 'path' can be absent if provided
+        without '/' at the start. Supposed that if 'path' is present than it
+        is prepended with '/'.
+        Removing '/' in the beginning to simplify usage in resource.
+        """
+        if data['op'] not in self.NO_VALUE_OPERATIONS and 'value' not in data:
+            raise ValidationError('value is required')
+
+        if 'path' not in data:
+            raise ValidationError('Path must always begin with /')
+        else:
+            data['field_name'] = data['path'][1:]
+
+    @classmethod
+    def perform_patch(cls, operations, obj, state=None):
+        """
+        Performs all necessary operations by calling class methods with corresponding names
+        """
+        for operation in operations:
+            if not cls._process_patch_operation(operation, obj=obj, state=state):
+                log.info("%s patching has ignored unknown operation %s", (obj.__name__, operation))
+                return False
+        return True
+
+    @classmethod
+    def _process_patch_operation(cls, operation, obj, state=None):
+        """
+        Args:
+            operation (dict) - one patch operation in RFC 6902 format.
+            obj (Team) - team instance which is needed to be patched.
+            state (dict) - inter-operations state storage
+
+        Returns:
+            processing_status (bool) - True if operation was handled, otherwise False.
+        """
+
+        field_name = operation['field_name']
+        field_value = operation['value']
```

Comment:
There is no point in extracting these here. It can even fail when `value` does not exist (when `op == 'remove'`).

Instead of extracting these here you would better extract `operation['op']` since otherwise you do this extraction in every `if` statement.